lib/bitcoin/BitcoinProcessor.ts

```diff
@@ -165,7 +165,7 @@ export default class BitcoinProcessor {
     await this.versionManager.initialize(versionModels, this.config, this.blockMetadataStore);
     await this.serviceStateStore.initialize();
     await this.blockMetadataStore.initialize();
-    await this.transactionStore.initialize(this.config.mongoDbConnectionString, this.config.databaseName);
```

Comment:
Ironically, reading this PR I just remembered that I was convinced a while back that taking params in `initialize()` is a better pattern pattern taking it in constructor. So if you agree, let's flip it around.